src/components/profile/profileView/ProfileView.jsx

```diff
@@ -43,7 +43,15 @@ export function ProfileView(props) {
   useEffect(() => {
     getProfile(user_id);
     getUserLikes(user_id)
-  }, [getProfile, getUserLikes, user_id, user_recipes]);
+  }, [getProfile, getUserLikes, user_id]);
+  // Problem: This hook, when conditional on user_recipes changing, was stuck in a loop. But how else to make sure the Forked Recipe
+  // and Forks counts (i.e. the profile) update when a recipe is (un)liked?
+  // Tentative solution: Remove user_recipes from the above dependency array, and make clicking the recipe container call
+  // getProfile(user_id) at a delay! It's imperfect... but removes the looping!
```

Comment:
have you tried an empty array? console warnings won't show on production

Reply:
For what, exactly? Need the profile to update each time a recipe is (un)liked. :)

Reply:
oh then if that's the case nevermind